msteams-platform/concepts/device-capabilities/native-device-permissions.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: Request device permissions for your Microsoft Teams tab
+title: Request device permissions for your Microsoft Teams app
 description: How to update your app manifest in order to request access to native features that usually require user consent
-keywords: teams tabs development
+keywords: teams apps development
 ---
 
-# Request device permissions for your Microsoft Teams tab
+# Request device permissions for your Microsoft Teams app
 
-You might want to enrich your tab with features that require access to native device functionality like:
+Enrich your app with features that require access to native device functionality, such as:
 
 > [!div class="checklist"]
 >
 > * Camera
 > * Microphone
+> * Gallery
 > * Location
-> * Notifications
 
-> [!NOTE]
-> To integrate camera and image capabilities within your Microsoft Teams mobile app, see [Camera and image capabilities in Teams.](../../concepts/device-capabilities/mobile-camera-image-permissions.md)
+[!NOTE] 
+* To integrate your native device media capabilities within Teams mobile app, see [Media capabilities in Teams.](mobile-camera-image-permissions.md)
+* The device permissions work in the same way for all app constructs, such as tabs or messaging extensions.
 
 > [!IMPORTANT]
 >
-> * At present, Teams mobile client only supports access to `camera`, `gallery`, `mic`, and `location` through native device capabilities and is available on all app constructs including tabs. </br>
-> * Support for `camera`, `gallery`, and `mic` is enabled through [**selectMedia API**](/javascript/api/@microsoft/teams-js/media?view=msteams-client-js-latest#selectMedia_MediaInputs___error__SdkError__attachments__Media_______void_&preserve-view=true). For single image capture you may use [**captureImage API**](/javascript/api/@microsoft/teams-js/microsoftteams?view=msteams-client-js-latest#captureimage--error--sdkerror--files--file-------void-&preserve-view=true).
+> * Support for `camera`, `gallery`, and `microphone` is enabled through [**selectMedia API**](/javascript/api/@microsoft/teams-js/media?view=msteams-client-js-latest#selectMedia_MediaInputs___error__SdkError__attachments__Media_______void_&preserve-view=true). For single image capture you may use [**captureImage API**](/javascript/api/@microsoft/teams-js/microsoftteams?view=msteams-client-js-latest#captureimage--error--sdkerror--files--file-------void-&preserve-view=true).
```

Comment:
@Rajeshwari-v, is there a reason why this important is here? Can we move it to a some other section? It is breaking the flow of the page. Lets discuss.